client/app/resources/i18n/email/Messages_fr_FR.xml

```diff
@@ -319,32 +319,58 @@
-   <translation id="email_tab_modal_migrate_configuration_guides_button" qtlid="436866">Cliquez ici pour accéder à nos guides techniques</translation>
+   <translation id="email_tab_modal_migrate_configuration_guides_button">Consulter nos guides techniques</translation>
    <translation id="email_tab_modal_migrate_aknowledge" qtlid="436879">En validant, je reconnais avoir pris connaissance de toutes les informations concernant la migration du compte e-mail source sélectionné.</translation>
-   <translation id="email_tab_modal_migrate_alert_not_supported" qtlid="436892">Attention, les éléments suivants ne seront pas pris en charge par la présente migration&#160;: <br />- vos filtres&#160;; <br />- vos répondeurs&#160;; <br />- vos mailing lists&#160;; <br />- vos contacts Roundcube.</translation>
+   <translation id="email_tab_modal_migrate_alert_not_supported">Les éléments suivants ne seront pas pris en charge par la migration&#160;:</translation>
+   <translation id="email_tab_modal_migrate_alert_not_supported_filters">vos filtres&#160;;</translation>
+   <translation id="email_tab_modal_migrate_alert_not_supported_responders">vos répondeurs&#160;;</translation>
+   <translation id="email_tab_modal_migrate_alert_not_supported_lists">vos mailing lists&#160;;</translation>
+   <translation id="email_tab_modal_migrate_alert_not_supported_roundcube_contacts">vos contacts Roundcube.</translation>
    <translation id="email_tab_modal_migrate_alert_configure_me" qtlid="436905">Il est nécessaire de disposer d’un compte e-mail non configuré (@configureme.me) pour réaliser l’opération.</translation>
-   <translation id="email_tab_modal_migrate_alert_dont_change_password" qtlid="444404">Attention, veuillez ne pas modifier le mot de passe de votre compte tant que sa migration n'est pas terminée.</translation>
-   <translation id="email_tab_modal_migrate_success" qtlid="436918">Votre compte e-mail est en cours de migration. L'opération sera effective d'ici 5 à 10 minutes.</translation>
+   <translation id="email_tab_modal_migrate_alert_dont_change_password">Veuillez ne pas modifier le mot de passe de votre compte e-mail source tant que sa migration n'est pas terminée.</translation>
+   <translation id="email_tab_modal_migrate_success_emailpro">La migration de votre compte e-mail vers votre service Email Pro est commencée.</translation>
+   <translation id="email_tab_modal_migrate_success_exchange">La migration de votre compte e-mail vers votre service Exchange est commencée.</translation>
    <translation id="email_tab_modal_migrate_error" qtlid="436931">Une erreur s'est produite lors de la migration du compte :</translation>
+   <translation id="email_tab_modal_migrate_error_check_label" >Une erreur s'est produite lors de la migration du compte{0} :</translation>
+   <translation id="email_tab_modal_migrate_errors_check_label" >Plusieurs erreurs se sont produites lors de la migration du compte{0} :</translation>
```

Comment:
`" >` => `">`
`compte{0}` => `compte {0}` instead?

Reply:
`compte{0}` : We concatenate with `email_tab_modal_migrate_error_check_should_retry` translation if we want to indicate to the user to wait before retry (optionnal, depends on error messages)